test/checkpoint/CMakeLists.txt

```diff
@@ -48,18 +48,19 @@ define_test_variants(test_checkpoint
         test_checkpoint
 )
 
-# FIXME-WT-15334 re-enable tests once the build is stable
```

Comment:
We can close [this](https://jira.mongodb.org/browse/WT-15334)?

Reply:
I will be closing the ticket and duping it to this one